### PR TITLE
tiny tweaks in CP/DP

### PR DESCRIPTION
**Pull request by hishamhm**

### CODE REVIEW

File: kong.conf.default [hunk 1]
```diff
@@ -124,6 +124,45 @@
...
 
+#------------------------------------------------------------------------------
+# HYBRID MODE
+#------------------------------------------------------------------------------
+
+#role = traditional              # Use this setting to enable Hybrid Mode,
+                                 # This allows running some Kong nodes in a
+                                 # control plane role with a database and
+                                 # have them deliver configuration updates
+                                 # other nodes running to DB-less running in
```
Comment (gszr) [inline kong.conf.default]:
```suggestion
                                 # to other nodes running to DB-less running in
```

File: kong.conf.default [hunk 2]
```diff
@@ -124,6 +124,45 @@
...
+                                 # when establishing secure communication
+                                 # between control and data plane nodes.
+                                 # You can use the `kong hybrid` command to
+                                 # generate the certificate/key pair.
+
+#cluster_cert_key =              # Filename of the cluster certificate key to use
+                                 # when establishing secure communication
+                                 # between control and data plane nodes.
+                                 # You can use the `kong hybrid` command to
+                                 # generate the certificate/key pair.
```
Comment (dndx) [inline kong.conf.default]: Also need:

```
#cluster_listen = 0.0.0.0:8005

                         # Comma-separated list of addresses and ports on
                         # which the cluster control plane (CP) server should listen for
                         # data plane connections.
                         # The cluster communication port of the
                         # control plane must be accessible by all the data planes
                         # within the same cluster. This port is mTLS protected
                         # to ensure end-to-end security and integrity.
                         #
                         # This setting has no effect if `role` is not set to
                         # `control_plane`
```

File: kong.conf.default (hunk 1 above)
Comment (hishamhm) [reply kong.conf.default]: thanks!

File: kong.conf.default (hunk 2 above)
Comment (hishamhm) [reply kong.conf.default]: thanks, added!

**Pull request merged**